interoperator/internal/properties/properties.go

```diff
@@ -31,7 +33,7 @@ type Status struct {
 	Provision   InstanceStatus `yaml:"provision" json:"provision"`
 	Bind        GenericStatus  `yaml:"bind" json:"bind"`
 	Unbind      GenericStatus  `yaml:"unbind" json:"unbind"`
-	Deprovision GenericStatus  `yaml:"deprovision" json:"deprovision"`
+	Deprovision InstanceStatus `yaml:"deprovision" json:"deprovision"`
```

Comment:
Maybe as a different PR, we need to document the structure of the expected rendered status template

https://github.com/cloudfoundry-incubator/service-fabrik-broker/blob/master/docs/Interoperator-templates.md#status

Reply:
As of now, empty values for new fields `instance_usable` and `update_repeatable` will be ignored. But I will modify the doc in next PR. Will list this item as pending in ticket.